src/main/java/com/balako/onlinebookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -35,6 +37,28 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(
         return new ResponseEntity<>(body, headers, status);
     }
 
+    @ExceptionHandler({RegistrationException.class})
+    protected ResponseEntity<Object> handleRegistrationException(
+            RegistrationException ex
+    ) {
+        Map<String, Object> body = new LinkedHashMap<>();
+        body.put("timestamp", LocalDateTime.now());
```

Comment:
You're creating the same map for each error response. I suggest to define the error response formate and create a separate class for error response.